controllers/update_status.go

```diff
@@ -629,3 +652,15 @@ func validateInstance(r *FoundationDBClusterReconciler, context ctx.Context, clu
 
 	return false, false, needsSidecarConfInConfigMap, nil
 }
+
+func cleanAddressList(slice []string) []string {
```

Comment:
Could you add a short comment here that we remove duplicates and and empty addresses?